timesys/high-level-triggers.py

```diff
@@ -17,6 +17,7 @@
             'Duration': {'value': 2000},
             'State': {'value': 0},
             'Polarity': {'value': 0, 'states': (0, 1)},
+            'Direction': {'value': 0, 'states': (0, 2)},
```

Comment:
Why is `states` set to that value? Does it have anything to do with the PV's possible values?

Reply:
They define the alarm levels of the PV. `0` means no alarm, `1` means minor alarm and `2` means major alarm. This implies that when the PV state is in its first enum value it will have no alarm, but when it is set to its second enum value it will have a major alarm. This is useful because pydm widgets will show a red frame around the PV when the alarm is on.
For more information, please refer to the pcaspy [documentation](https://pcaspy.readthedocs.io/en/latest/api.html#simpleserver).